CHANGES.md

```diff
@@ -5,10 +5,12 @@
 ### Bug Fixes
 
 * filter: Improved warning and error messages in the case of missing columns. [#1604] (@victorlin)
+* filter: Properly error on duplicates in `--sequences`. [#1613] (@victorlin)
```

Comment:
This is ostensibly a bug fix but is likely a backwards-incompatible breaking change for lots of existing usages. I'd advocate for considering it a breaking change.

Reply:
+1 on treating this as a breaking change from me; new errors for old working behavior deserves it.

Reply:
Updated in [force-push](https://github.com/nextstrain/augur/compare/e9e31f595f88aba8cb9e29ab4604da3ea0dc8b06..ab967fd7ae8c786b87d844985761f3e5094e08fa), added https://github.com/nextstrain/augur/labels/breaking label